src/nni_manager/training_service/reusable/trialDispatcher.ts

```diff
@@ -664,17 +664,16 @@ class TrialDispatcher implements TrainingService {
     }
 
     private releaseEnvironment(trial: TrialDetail): void {
-        if (undefined === trial.environment) {
-            throw new Error(`TrialDispatcher: environment is not assigned to trial ${trial.id}, and cannot be released!`);
```

Comment:
What's reason to remove the check? it helps to find unexpected behavior

Reply:
IT find a case that assessor report two kill command continuous for a trial, a trial released environment, then release again will throw exception.